build/Makefile.vars.common

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020-2021 The CapableVMs "CHERI Examples" Contributors.
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+# Common configuration for all 'Make' targets.
+# This should not be invoked directly.
+
+# Set by platform-specific makefiles. For example: ~/cheri/output/sdk
+ifndef SDKBASE
+$(error SDKBASE is not set)
+endif
+
+# For example: morello-purecap
+ifndef PLATFORM
+$(error PLATFORM is not set)
+endif
+
```

Comment:
Should we add `ifndef SSHPORT` here?

Reply:
Ah, I see that elsewhere we do `ifdef SSHPORT`. It probably would make sense to do all of the checks in one place (presumably here), but perhaps that's for another PR?

Reply:
I decided against it because it allows you to build everything without having a sensible value for `SSHPORT` (e.g. because you don't have a model running).